app/utilities/schema.py

```diff
@@ -45,18 +46,18 @@ def load_schema_file(schema_file, language_code=None):
         raise e
 
 
-def _get_schema_file_path(schema_file, language_code):
-    if language_code != 'en':
-        schema_dir = os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
-    else:
-        schema_dir = settings.EQ_SCHEMA_DIRECTORY
+def _get_schema_path(language_code='en'):
+    return os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
 
+
+def _get_schema_file_path(schema_file, language_code):
+    schema_dir = _get_schema_path(language_code)
     return os.path.join(schema_dir, schema_file)
 
 
-def available_json_schemas():
+def available_json_schemas(language_code='en'):
     files = []
-    for file in os.listdir(settings.EQ_SCHEMA_DIRECTORY):
-        if os.path.isfile(os.path.join(settings.EQ_SCHEMA_DIRECTORY, file)) and file.endswith(".json"):
+    for file in os.listdir(_get_schema_path(language_code)):
+        if os.path.isfile(_get_schema_file_path(file, language_code)) and file.endswith(".json"):
```

Comment:
You shouldn't need to pass `language_code` to either of these methods as they should default to `DEFAULT_LANGUAGE_CODE`